internal/sapcontrol/webservice.go

```diff
@@ -269,10 +306,10 @@ func (service *webService) HACheckConfig() (*HACheckConfigResponse, error) {
 }
 
 // implements WebService.HACheckFailoverConfig()
-func (service *webService) HACheckFailoverConfig() (*HACheckFailoverConfigResponse, error) {
+func (s *webService) HACheckFailoverConfig() (*HACheckFailoverConfigResponse, error) {
```

Comment:
if this is autogenerated code how can it be that it changes?

Reply:
This is not auto-generated, some of its content was copied from the autogenerated stub and then modified. The file is to be treated like a normal source file.